test/test_suites.py

```diff
@@ -1,6 +1,5 @@
 from collections import OrderedDict
 from test_suit.test_string_json import TestStringJson
-from test_suit.test_string_json_proxy import TestStringJsonProxy
```

Comment:
Any reason this is removed?

Reply:
Unused import.

Reply:
proxy test is added in test_verify towards the end which I recently enabled.